README.md

```diff
@@ -48,10 +43,15 @@ $ $GOPATH/bin/terraform-provider-aws
 ...
 ```
 
+Using the Provider
+----------------------
+
+If you're building the provider, follow the instructions to [install it as a plugin.](https://www.terraform.io/docs/plugins/basics.html#installing-a-plugin) After placing it into your plugins directory,  run `terraform init` to initialize it. Documentation about the provider specific configuration options can be found on the [provider's website](https://www.terraform.io/docs/providers/aws/index.html).
```

Comment:
I wonder if we should adjust this section to include "normal" usage or update the heading as well just to remove any potential confusion. e.g.

```
Using the Provider
----------------------

To use a released provider in your Terraform environment, run [`terraform init`](https://www.terraform.io/docs/commands/init.html) and Terraform will automatically install the provider. To specify a particular provider version when installing released providers, see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#version-provider-versions).

To instead use a custom built provider in your Terraform environment (e.g. the provider binary from the build instructions above), follow the instructions to [install it as a plugin](https://www.terraform.io/docs/plugins/basics.html#installing-a-plugin). After placing it into your plugins directory,  run `terraform init` to initialize it.

For either installation method, documentation about the provider specific configuration options can be found on the [provider's website](https://www.terraform.io/docs/providers/aws/index.html).
```

Or:

```
Using the Custom Built Provider
----------------------
```

Reply:
I like the former as it's less confusing. If I saw the heading _Using the Custom Built Provider_ I would start freaking out.